### DISCUSSION TITLE
请问如何修改Pagination 条/页 文案为 规则/页

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#49493] [Q&A]
如题

**Comment by afc163** [comment]
From 千问：

在Ant Design（antd）中自定义Pagination组件的文案，如将默认的"条/页"改为"规则/页"，可以通过覆盖Pagination组件的文案属性实现。具体做法如下：

Ant Design 提供了`locale`属性来国际化和自定义组件内的文案。对于Pagination组件，你可以通过创建一个自定义的locale对象并将其传递给Pagination组件来改变“条/页”这一文案。

以下是一个简单的示例代码：

```jsx
import React from 'react';
import { Pagination } from 'antd';

// 自定义locale对象
const customPaginationLocale = {
  // ... 可以在此处添加其他需要自定义的文案
  items_per_page: '规则/页', // 修改"条/页"为"规则/页"
};

function App() {
  return (
    <div>
      <Pagination
        total={500}
        pageSize={10}
        locale={customPaginationLocale} // 应用自定义的locale
      />
    </div>
  );
}

export default App;
```

请注意，上述代码中的`items_per_page`字段是用来替换“条/页”的文案的。如果你使用的是较新版本的Ant Design，可能需要查阅最新的文档确认文案对应的键名是否有变化，因为随着版本迭代，具体的字段名可能会有所不同。不过，基本思路就是通过覆盖locale对象中的相应键值来达到自定义文案的目的。
